lib/aggregate/attribute/hash.rb

```diff
@@ -58,6 +70,16 @@ def convert_to_json(value)
           ActiveSupport::JSON.encode(value || DEFAULT_VALUE)
         end
       end
+
+      def should_store_hash_as_json
+        if options.dig(:aggregate_db_storage_type) == :elasticsearch
+          false
+        elsif options.key?(:store_hash_as_json)
```

Comment:
Similar question: `:store_hash_as_json` sounds more specific so should we take that value if it exists?